server/config/options.go

```diff
@@ -51,19 +54,25 @@ type CdcConfig struct {
 }
 
 type TracingConfig struct {
-	Enabled           bool
-	WithUDS           string
-	WithAgentAddr     string
-	WithDogStatsdAddr string
+	Enabled           bool   `mapstructure:"enabled" yaml:"enabled" json:"enabled"`
+	WithUDS           string `mapstructure:"agent_socket" yaml:"agent_socket" json:"agent_socket"`
+	WithAgentAddr     string `mapstructure:"agent_addr" yaml:"agent_addr" json:"agent_addr"`
+	WithDogStatsdAddr string `mapstructure:"dogstatsd_addr" yaml:"dogstatsd_addr" json:"dogstatsd_addr"`
 }
 
 type ProfilingConfig struct {
-	Enabled         bool
-	EnableCPU       bool
-	EnableHeap      bool
-	EnableBlock     bool
-	EnableMutex     bool
-	EnableGoroutine bool
+	Enabled         bool `mapstructure:"enabled" yaml:"enabled" json:"enabled"`
+	EnableCPU       bool `mapstructure:"enable_cpu" yaml:"enable_cpu" json:"enable_cpu"`
```

Comment:
I'm not sure if these are necessary when there are no significant changes to the pattern?

Reply:
I don't think we need all these knobs here if it is enabled then in most of the cases we want to profile different things like CPU, Heap, etc. Just have one config key Enabled.